src/app/pages/list/sortable-list/sortable-list.component.scss

```diff
@@ -0,0 +1,42 @@
+mat-nav-list {
+    padding-top: 0px;
+}
+
+.mat-list-base .mat-list-item {
+    height: 56px;
+}
+
+// the floating component when dragging
+.cdk-drag-preview {
```

Comment:
Are these still needed after switching to infolistitem? Just curious...I know we had a lot of annoying issues with the preview/placeholder styles before.

Reply:
We didn't need them and getting rid of them actually fixed a style issue I was seeing.

Reply:
We still need these for the proper UX/transitions/animations